tests/test_events/test_events_ratios.py

```diff
@@ -60,6 +64,35 @@ def test_event_checkin_and_cancellation_ratio_per_room(self, mock_get_json):
             event_ratio_per_room_response
         )
 
+    @patch("helpers.calendar.analytics_helper.get_events_within_datetime_range",
+           spec=True)
+    def test_event_checkin_and_cancellation_single_room(self, mock_get_json):
+        """
+        Test that an admin is able to get the ratio of checkins to bookings
+        for each individual room
+        """
+        mock_get_json.return_value = get_events_mock_data()
+        CommonTestCases.admin_token_assert_equal(
+            self,
+            event_ratio_single_room_query,
+            event_ratio_single_room_response
+        )
+
+    @patch("helpers.calendar.analytics_helper.get_events_within_datetime_range",
+           spec=True)
+    def test_event_checkin_cancellation_single_room_wrong_id(self,
+                                                             mock_get_json):
+        """
+        Test that an admin is able to get the ratio of checkins to bookings
```

Comment:
@jesseinit This docstring is the same as the one on line 71, kindly change accordingly.

Reply:
Fixed